pkg/engine/lifecycletest/parameterized_test.go

```diff
@@ -272,10 +291,10 @@ func TestReplacementParameterizedProvider(t *testing.T) {
 		p.GetProject(), p.GetTarget(t, nil), p.Options, false, p.BackendClient, nil, "up")
 	require.NoError(t, err)
 	assert.NotNil(t, snap)
-	assert.Len(t, snap.Resources, 5)
+	assert.Len(t, snap.Resources, 7)
 
-	// Check that we loaded the provider twice
-	assert.Equal(t, 2, loadCount)
+	// Check that we loaded the provider thrice
```

Comment:
I really hope we don't have to load it more times, unless we want to be edgy and use `frice`, `quince`, etc.